mcs/class/corlib/System/MonoCustomAttrs.cs

```diff
@@ -149,7 +149,15 @@ internal static object[] GetCustomAttributes (ICustomAttributeProvider obj, Type
 			if (obj == null)
 				throw new ArgumentNullException ("obj");
 			if (attributeType == null)
-				throw new ArgumentNullException ("attributeType");	
+				throw new ArgumentNullException ("attributeType");
+			
+#if NETCORE
+			if (!attributeType.IsSubclassOf(typeof(MonoCustomAttrs)) && 
```

Comment:
Do we really need both of these classes ?

Reply:
Yeah, `MonoCustomAttrs` check is redundant, removed.